easybuild/scripts/bootstrap_eb.py

```diff
@@ -659,14 +659,36 @@ def stage2(tmpdir, templates, install_path, distribute_egg_dir, sourcepath):
         'preinstallopts': preinstallopts,
     })
 
-    # create easyconfig file
-    ebfile = os.path.join(tmpdir, 'EasyBuild-%s.eb' % templates['version'])
-    handle = open(ebfile, 'w')
+    # need to initialise build options before pypi_source_urls function can be used
+    from test.framework.utilities import init_config
+    init_config()
```

Comment:
I'm not entirely happy with this bit, but I consider the stuff below a temporary fix.

Once EasyBuild v3.6.0 is out that includes the fix for `derive_alt_pypi_url`, we can roll back this change.